pkg/repository/manifest/repo.go

```diff
@@ -16,16 +16,32 @@ package manifest
 import (
 	"encoding/json"
 	"fmt"
-	"io"
+	"io/ioutil"
 	"time"
 
-	cjson "github.com/gibson042/canonicaljson-go"
 	"github.com/pingcap-incubator/tiup/pkg/repository/crypto"
-	"github.com/pingcap/errors"
 )
 
-// Init creates and initializes an empty reposityro
-func Init(dst string, initTime time.Time) error {
+// Init creates and initializes an empty repository
+func Init(dst string, initTime time.Time, pub, priv string) error {
+	// read key files
+	privBytes, err := ioutil.ReadFile(priv)
```

Comment:
we should not take keys from the user, instead we generate them ourselves and print the private key

Reply:
This is intended for easier local developing and testing, key generating and storing will be added in another PR that implement `genkey` subcommand.